Grid Header Template fix and kb linking (#388)
telerik/blazor-docs: knowledge-base/grid-center-column-header-content.md

}
````

## Notes

> * If you need to wrap the column header content as well, you can also try the approach demonstrated in this knowledge base article - [Wrap and center the Grid Column Header text]({%slug grid-kb-wrap-and-center-column-header-text%}). It showcases how to change the default display property of the header cells to `block` and then easily operate with their content to wrap and center it.

Reviewer: ```suggestion
* If you need to wrap the column header content as well, you can try the approach from this knowledge base article - [Wrap and center the Grid Column Header text]({%slug grid-kb-wrap-and-center-column-header-text%}). It shows how to change the default display property of the header cells to `block` and then easily operate with their content to wrap and center it.
```

> * If you want full control over the header text contents and rendering, you can use the [column header template]({%slug grid-templates-column-header%}).

Reviewer: ```suggestion
* If you want full control over the header text contents and rendering, you can use a [column header template]({%slug grid-templates-column-header%}).
```
